.github/workflows/js-tests.yml

```diff
@@ -1,12 +1,19 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+# Copyright (C) 2025 KTH Royal Institute of Technology.
+
 name: CI
 
 on:
   push:
     branches:
       - master
+      - maint-*
```

Comment:
i am not sure, but in the other packages we did a "- maint-**", so with double quotes. i am not sure if this works